EligibilityCheckOrchestrator/index.ts

```diff
@@ -37,6 +38,27 @@ export type OrchestratorInput = t.TypeOf<typeof OrchestratorInput>;
 
 const NOTIFICATION_DELAY_SECONDS = 10;
 
+export const getMessage = (
+  messageType: keyof typeof MESSAGES,
+  validBefore: Date
+): MessageContent => {
+  switch (messageType) {
+    case "EligibilityCheckSuccessEligible":
+    case "EligibilityCheckSuccessEligibleWithDiscrepancies":
+      return MESSAGES[messageType](validBefore);
+    case "EligibilityCheckSuccessIneligible":
+    case "EligibilityCheckFailure":
+    case "EligibilityCheckConflict":
+    case "BonusActivationSuccess":
+    case "BonusActivationFailure":
+      return MESSAGES[messageType]();
+    default:
+      throw new Error(
```

Comment:
I suggest the use of `assertNever` so we can be warned at compile time for a missing case.

Reply:
done